aten/CMakeLists.txt

```diff
@@ -337,6 +341,9 @@ int main()
 }
 " HAS_C11_ATOMICS)
 
+# For ROCm stack, the hcc/hipcc compiler cannot use C11 atomics. Prefer GCC.
+SET(HAS_C11_ATOMICS OFF)
```

Comment:
This SET() is being applied unconditionally, even if you're not using the ROCM stack.